docs/manual/working/scalaGuide/main/json/ScalaJson.md

```diff
@@ -0,0 +1,186 @@
+<!--- Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com> -->
+# JSON basics
+
+Modern web applications often need to parse and generate data in the JSON (JavaScript Object Notation) format. Play supports this via its [JSON library](https://oss.sonatype.org/service/local/repositories/public/archive/com/typesafe/play/play-json_2.12/2.6.0-M1/play-json_2.12-2.6.0-M1-javadoc.jar/!/play/api/libs/json/).
```

Comment:
The Sonatype URL is fine for me, as updated on each publish, but having a variable for the latest would be nice in this case